code/game/turfs/turf.dm

```diff
@@ -108,12 +108,14 @@
 /turf/attackby(obj/item/C, mob/user, params)
 	if(..())
 		return TRUE
+	var/list/contents_cached = contents
 	if(can_lay_cable() && istype(C, /obj/item/stack/cable_coil))
```

Comment:
You're creating a cached version of this var even if this `if` never passes, seems wasteful.

Also you could likely swap the order here. iirc istype() overhead is less than proc call overhead, you'd want the

Reply:
Will do